### PR TITLE
[SWE-1475] [BRANDBOOK] Check fontFamily in Alphabet, Make icon screen

### PR DESCRIPTION
**Pull request by AnastasiaShapowalova**
Screenshot 2022-07-26 at 11 36 53

Screenshot 2022-07-26 at 16 55 13

### CODE REVIEW

**Review by olenashevchuk — APPROVED** [review]

File: src/app/screens/Typography/TypographyContent/SecondaryScreen.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      subtitle="Semibold / Medium / Regular"
+      chapter="Secondary"
+      domainName="Typo."
+      // TODO get font family from  vars
+      content={
+        <Box mt={4}>
+          <Alphabet />
+        </Box>
+      }
+      pageNumber={'09'}
```
Comment (olenashevchuk) [inline src/app/screens/Typography/TypographyContent/SecondaryScreen.js:19]: Delete {}

File: src/styles/vars/index.css [hunk 2]
```diff
@@ -9,4 +9,5 @@
 @import url('./alphabet.css');
 @import url('./input.css');
 @import url('./chapter.css');
-@import url('./iconView.css');
+@import url('./iconography.css');
+@import url('./contentScreen.css');
```
Comment (olenashevchuk) [inline src/styles/vars/index.css:13]: see above

**Review by olenashevchuk — CHANGES_REQUESTED** [review]

**Review by olenashevchuk — APPROVED** [review]

File: src/app/components/bundles/ContentScreen/ContentScreen.js [hunk 3]
```diff
@@ -2,10 +2,11 @@ import {
...
   ContentScreenInfo
-} from 'components/bundles/ContentScreen/components'
+} from '~/app/components/bundles/ContentScreen/components'
 import { Footer, Header } from 'components'
 
 import ContentScreenStyled from './ContentScreen.styled'
+import PropTypes from 'prop-types'
 import React from 'react'
 
 const ContentScreen = (props) => {
```
Comment (Olexandr25) [inline src/app/components/bundles/ContentScreen/ContentScreen.js:12]: try to leave here comment - js doc

File: src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js [hunk 4]
```diff
@@ -1,12 +1,38 @@
+import {
+  DEFAULT_SCREEN_CONTENT_COLUMN_WIDTH,
+  DEFAULT_SCREEN_CONTENT_OFFSET
+} from 'components/bundles/ContentScreen/__constants__'
+
 import { Col } from 'components'
 import PropTypes from 'prop-types'
 
 const ContentScreenContent = (props) => {
```
Comment (Olexandr25) [inline src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js:9]: comment - js doc

File: src/app/components/bundles/ContentScreen/components/ContentScreenInfo/ContentScreenInfo.js [hunk 5]
```diff
@@ -1,10 +1,12 @@
 import { Box, Col, Divider, Text, Title } from 'components'
 
+import PropTypes from 'prop-types'
+
 const ContentScreenInfo = (props) => {
```
Comment (Olexandr25) [inline src/app/components/bundles/ContentScreen/components/ContentScreenInfo/ContentScreenInfo.js:5]: comment - js doc

File: src/app/components/elements/Alphabet/Alphabet.js [hunk 6]
```diff
@@ -1,21 +1,21 @@
+import { ALPHABET } from 'app/constants'
 import { AlphabetStyled } from './Alphabet.styled'
 import PropTypes from 'prop-types'
 import { Text } from '~/app/components'
 
 const Alphabet = (props) => {
```
Comment (Olexandr25) [inline src/app/components/elements/Alphabet/Alphabet.js:6]: comment - js doc

File: src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamples.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import { ContentScreen } from '~/app/components'
+import IconList from '~/app/domains/Iconography/components/IconList'
+import React from 'react'
```
Comment (Olexandr25) [inline src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamples.js:3]: do you need here React?

File: src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamples.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+const IconographyExamples = () => {
```
Comment (Olexandr25) [inline src/app/screens/Iconography/IconographyContent/IconographyExamples/IconographyExamples.js:5]: comment - js doc

File: src/app/screens/Typography/TypographyContent/SecondaryScreen.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import { Alphabet } from 'app/components'
+import { Box } from 'components'
+import { ContentScreen } from 'components'
+
+const SecondaryScreen = () => {
```
Comment (Olexandr25) [inline src/app/screens/Typography/TypographyContent/SecondaryScreen.js:5]: comment - js doc

File: src/app/components/elements/Alphabet/Alphabet.js [hunk 10]
```diff
@@ -1,21 +1,21 @@
...
-  const { alphabet, fontFamily } = props
+  const { fontFamily } = props
 
   return (
     <AlphabetStyled>
       <Text variant="brand-alphabet" fm={fontFamily}>
```
Comment (whoiam460) [inline src/app/components/elements/Alphabet/Alphabet.js:11]: first merge development inside this branch pr dev have important changes

**Pull request closed without merging**